src/clj_momo/lib/es/document.clj

```diff
@@ -147,6 +147,21 @@
       safe-es-read
       :found))
 
+(defn sort-params
+  [sort_by sort_order]
+  (let [sort-field-str (if (keyword? sort_by)
+                         (name sort_by)
+                         sort_by)
```

Comment:
I'm not sure what sort_by may be, but it is either string or keyword, you could choose to always call name on it.  'name is safe for strings.